### PR TITLE
add codmirror show-invisibles addon

### LINKED ISSUE #66: Show Invisibles
can you add the following plugin?

https://github.com/coderaiser/cm-show-invisibles

### PR DESCRIPTION
**Pull request by lisps**
i have added the addon https://github.com/coderaiser/cm-show-invisibles

grafik

Closes #66

### CODE REVIEW

File: lang/en/lang.php [hunk 1]
```diff
@@ -9,3 +9,4 @@
 $lang['js']['setting_nativeeditor'] = 'Native DokuWiki editor';
 $lang['js']['setting_syntax'] = 'Highlight syntax';
 $lang['js']['setting_theme'] = 'Color theme';
+$lang['js']['showinvisibles'] = 'Show Whitespace';
```
Comment (albertgasset) [inline lang/en/lang.php]: The key of the string should be "setting_showinvisibles"

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "grunt-contrib-jshint": "^1.1.0",
     "grunt-contrib-less": "^1.4.0",
     "grunt-contrib-uglify": "^2.0.0",
-    "grunt-contrib-watch": "^1.0.0"
-  }
+    "grunt-contrib-watch": "^1.0.0",
+    "cm-show-invisibles": "^1.2.1"
+  },
```
Comment (albertgasset) [inline package.json]: The trailing comma in line 12 causes `npm install` to fail.

**Comment by lisps** [comment]
Thanks for your hints. I corrected them.

**Comment by lisps** [comment]
can you create a new version, please?

**Pull request merged**